examples/rustapi_module/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools_rust>=0.10.2"]
```

Comment:
Travis is missing hypothesis, I guess it's missing here or up in the .travis.yml

Reply:
This is only for the build system. `tox` should install it as part of the `deps` because it's in `requirements-dev.txt`. I'll figure out what's going on.

Reply:
Oh I see the problem, it's the `for example in examples/*` stuff.